src/dlgPackageExporter.cpp

```diff
@@ -984,7 +1002,7 @@ dlgPackageExporter::zipPackage(const QString& stagingDirName, const QString& pac
     }
 
     if (isOk) {
-        zip_set_archive_comment(archive, packageConfig.toUtf8().constData(), packageConfig.toUtf8().length());
+        zip_set_archive_comment(archive, packageComment.toUtf8().constData(), static_cast<zip_uint16_t>(packageComment.toUtf8().length()));
```

Comment:
Didn't the previous, simpler construct work? Would rather keep it simple.

Reply:
Well, `packageConfig` != `packageComment` now - or do you mean the addition of the `static_cast<zip_uint16_t>` to eliminate a warning because `libzip` expects an unsigned 16 bit integer value whereas the Qt `QByteArray::length()` produces an `int` value?